<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Editor</class>
 <widget class="QMainWindow" name="Editor">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>954</width>
    <height>464</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>DynaMap Editor</string>
  </property>
  <widget class="QWidget" name="centralWidget">
   <layout class="QVBoxLayout" name="verticalLayout">
    <item>
     <widget class="QTabWidget" name="tabWidget">
      <property name="currentIndex">
       <number>1</number>
      </property>
      <widget class="QWidget" name="SoftwareEditor">
       <attribute name="title">
        <string>Softwares</string>
       </attribute>
       <layout class="QVBoxLayout" name="verticalLayout_2">
        <item>
         <widget class="QTreeView" name="SoftwareTree"/>
        </item>
        <item>
         <layout class="QHBoxLayout" name="horizontalLayout">
          <property name="topMargin">
           <number>10</number>
          </property>
          <item>
           <widget class="QPushButton" name="SoftAdd">
            <property name="text">
             <string>Add Software</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QPushButton" name="SoftDel">
            <property name="text">
             <string>Remove Software</string>
            </property>
           </widget>
          </item>
          <item>
           <spacer name="horizontalSpacer_3">
            <property name="orientation">
             <enum>Qt::Horizontal</enum>
            </property>
            <property name="sizeHint" stdset="0">
             <size>
              <width>40</width>
              <height>20</height>
             </size>
            </property>
           </spacer>
          </item>
          <item>
           <widget class="QPushButton" name="SoftGroupAdd">
            <property name="text">
             <string>Add Group</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QPushButton" name="SoftGroupDel">
            <property name="text">
             <string>Remove Group</string>
            </property>
           </widget>
          </item>
          <item>
           <spacer name="horizontalSpacer">
            <property name="orientation">
             <enum>Qt::Horizontal</enum>
            </property>
            <property name="sizeHint" stdset="0">
             <size>
              <width>40</width>
              <height>20</height>
             </size>
            </property>
           </spacer>
          </item>
          <item>
           <widget class="QPushButton" name="SoftInstanceAdd">
            <property name="text">
             <string>Add Instance</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QPushButton" name="SoftInstanceDel">
            <property name="text">
             <string>Remove Instance</string>
            </property>
           </widget>
          </item>
          <item>
           <spacer name="horizontalSpacer_2">
            <property name="orientation">
             <enum>Qt::Horizontal</enum>
            </property>
            <property name="sizeHint" stdset="0">
             <size>
              <width>40</width>
              <height>20</height>
             </size>
            </property>
           </spacer>
          </item>
          <item>
           <widget class="QPushButton" name="SoftItemAdd">
            <property name="text">
             <string>Add Item</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QPushButton" name="SoftItemDel">
            <property name="text">
             <string>Remove Item</string>
            </property>
           </widget>
          </item>
         </layout>
        </item>
       </layout>
      </widget>
      <widget class="QWidget" name="ControllerEditor">
       <attribute name="title">
        <string>Controllers</string>
       </attribute>
       <layout class="QVBoxLayout" name="verticalLayout_3">
        <item>
         <layout class="QHBoxLayout" name="horizontalLayout_3">
          <property name="leftMargin">
           <number>0</number>
          </property>
          <item>
           <widget class="QWidget" name="ControllerUI" native="true">
            <property name="sizePolicy">
             <sizepolicy hsizetype="Expanding" vsizetype="Expanding">
              <horstretch>0</horstretch>
              <verstretch>0</verstretch>
             </sizepolicy>
            </property>
            <layout class="QVBoxLayout" name="verticalLayout_4"/>
           </widget>
          </item>
          <item>
           <widget class="QTreeView" name="ControllerTree">
            <property name="sizePolicy">
             <sizepolicy hsizetype="MinimumExpanding" vsizetype="Expanding">
              <horstretch>0</horstretch>
              <verstretch>0</verstretch>
             </sizepolicy>
            </property>
            <property name="minimumSize">
             <size>
              <width>300</width>
              <height>0</height>
             </size>
            </property>
            <attribute name="headerCascadingSectionResizes">
             <bool>true</bool>
            </attribute>
           </widget>
          </item>
         </layout>
        </item>
        <item>
         <layout class="QHBoxLayout" name="horizontalLayout_2">
          <property name="topMargin">
           <number>0</number>
          </property>
          <item>
           <widget class="QPushButton" name="ControllerSubdivision">
            <property name="text">
             <string>Manage SubDivision</string>
            </property>
           </widget>
          </item>
          <item>
           <spacer name="horizontalSpacer_4">
            <property name="orientation">
             <enum>Qt::Horizontal</enum>
            </property>
            <property name="sizeHint" stdset="0">
             <size>
              <width>40</width>
              <height>20</height>
             </size>
            </property>
           </spacer>
          </item>
          <item>
           <widget class="QPushButton" name="ControllerBank">
            <property name="text">
             <string>Manage Bank</string>
            </property>
           </widget>
          </item>
          <item>
           <spacer name="horizontalSpacer_5">
            <property name="orientation">
             <enum>Qt::Horizontal</enum>
            </property>
            <property name="sizeHint" stdset="0">
             <size>
              <width>40</width>
              <height>20</height>
             </size>
            </property>
           </spacer>
          </item>
          <item>
           <widget class="QPushButton" name="ControllerControls">
            <property name="text">
             <string>Manage Controls</string>
            </property>
           </widget>
          </item>
         </layout>
        </item>
       </layout>
      </widget>
      <widget class="QWidget" name="MappingEditor">
       <attribute name="title">
        <string>Mapping</string>
       </attribute>
      </widget>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QMenuBar" name="menuBar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>954</width>
     <height>27</height>
    </rect>
   </property>
   <widget class="QMenu" name="menuFile">
    <property name="title">
     <string>File</string>
    </property>
    <addaction name="actionOpen_Software_file"/>
    <addaction name="actionSave_Software_File"/>
    <addaction name="actionSave_Selection"/>
    <addaction name="separator"/>
    <addaction name="actionOpen_Controller_UI"/>
    <addaction name="actionOpen_Controller_file"/>
    <addaction name="actionSave_Controller_File"/>
    <addaction name="separator"/>
    <addaction name="actionOpen_Mapping_File"/>
    <addaction name="actionSave_Mapping_File"/>
   </widget>
   <widget class="QMenu" name="menuView">
    <property name="title">
     <string>View</string>
    </property>
    <addaction name="actionHide_Options"/>
    <addaction name="actionShow_Options_for_Selected"/>
    <addaction name="separator"/>
    <addaction name="actionHide_Groups"/>
   </widget>
   <addaction name="menuFile"/>
   <addaction name="menuView"/>
  </widget>
  <widget class="QToolBar" name="mainToolBar">
   <attribute name="toolBarArea">
    <enum>TopToolBarArea</enum>
   </attribute>
   <attribute name="toolBarBreak">
    <bool>false</bool>
   </attribute>
  </widget>
  <widget class="QStatusBar" name="statusBar"/>
  <action name="actionOpen_Software_file">
   <property name="text">
    <string>Open Software file</string>
   </property>
  </action>
  <action name="actionSave_Software_File">
   <property name="text">
    <string>Save Software File</string>
   </property>
  </action>
  <action name="actionOpen_Controller_file">
   <property name="text">
    <string>Open Controller File</string>
   </property>
  </action>
  <action name="actionSave_Controller_File">
   <property name="text">
    <string>Save Controller File</string>
   </property>
  </action>
  <action name="actionOpen_Mapping_File">
   <property name="text">
    <string>Open Mapping File</string>
   </property>
  </action>
  <action name="actionSave_Mapping_File">
   <property name="text">
    <string>Save Mapping File</string>
   </property>
  </action>
  <action name="actionHide_Options">
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="text">
    <string>Hide Options</string>
   </property>
  </action>
  <action name="actionShow_Options_for_Selected">
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="text">
    <string>Show Options for Selected</string>
   </property>
  </action>
  <action name="actionHide_Groups">
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="text">
    <string>Hide Groups</string>
   </property>
  </action>
  <action name="actionSave_Selection">
   <property name="text">
    <string>Save Selection</string>
   </property>
  </action>
  <action name="actionOpen_Controller_UI">
   <property name="text">
    <string>Open Controller UI</string>
   </property>
  </action>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources/>
 <connections/>
</ui>
